add benchmark (#650)
stac-utils/stac-fastapi: CHANGES.md

## [Unreleased]

### Changed

> * Make sure FastAPI use Pydantic validatio and serialization by no wrapping endpoint output with a Response object ([#650](https://github.com/stac-utils/stac-fastapi/pull/650))

Reviewer: ```suggestion
* Make sure FastAPI uses Pydantic validation and serialization by not wrapping endpoint output with a Response object ([#650](https://github.com/stac-utils/stac-fastapi/pull/650))
```
